E2-Breakdown-Money/src/main/java/com/redelles/e2/Utils.java

```diff
@@ -4,20 +4,71 @@
 
 public class Utils {
 
-    public static Integer getNumber(){
+    public static Integer getAmount(){
+
+        Integer amount = -1;
+        Boolean flag;
+
+        do{
+            try {
+                System.out.println("Please enter an amount of money.");
+                amount = Utils.scanNumber();
+                flag = false;
+            } catch (Exception e) {
+                System.out.println("REMEMBER This application only works with numbers.");
+                flag = true;
+            }
+
+        } while (flag);
```

Comment:
Instead of to use a new variable flag you can check if the amount is > -1 or > 0 depends if you want to have a 0 as valid amount